### PR TITLE
Unskip Mini Cart i18n test

### LINKED ISSUE #45346: [Mini Cart] E2E tests: tests regarding translations are failing
During the work on #45324, we noticed that the E2E tests regarding the Mini Cart block and translations were starting to fail. As a workaround, we disabled them, but we should double-check why this happens.

```
    [blockThemeWithGlobalSideEffects] › mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts:25:6 › Shopper → Translations › User can see translation in empty Mini-Cart 
    [blockThemeWithGlobalSideEffects] › mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts:43:6 › Shopper → Translations › User can see translation in filled Mini-Cart 
```

### PR DESCRIPTION
**Pull request by nielslange**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #45346.

This PR contains the following changes:

- Unskip test `User can see translation in empty Mini-Cart`.
- Install language packs in `plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts`.
- Install language packs in `plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts`.
- Delete `plugins/woocommerce-blocks/tests/e2e/bin/scripts/parallel/translations.sh` as language pcks will be installed directly in the previous mentioned files.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Verify that all CI tests are passing.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts [hunk 1]
```diff
@@ -21,13 +21,22 @@ const test = base.extend< { checkoutPageObject: CheckoutPage } >( {
...
+			`npm run wp-env run tests-cli -- wp language core install nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp site switch-language nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin install woocommerce nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin update woocommerce`
```
Comment (gigitux) [inline plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts]: Is there a particular reason why we run this command?

File: plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts [hunk 2]
```diff
@@ -12,19 +12,26 @@ import { REGULAR_PRICED_PRODUCT_NAME } from '../checkout/constants';
...
+			`npm run wp-env run tests-cli -- wp language core install nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp site switch-language nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin install woocommerce nl_NL`
+		);
+		await cli(
+			`npm run wp-env run tests-cli -- wp language plugin update woocommerce`
```
Comment (gigitux) [inline plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts]: Same here

**Review by gigitux — APPROVED** [review]
Thanks for your work! LGTM! 🙇

I'm pre-approving the PR, but I left a comment. 

```
npm run wp-env run tests-cli -- wp language plugin update woocommerce
```

Is this command necessary?

File: plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts (hunk 1 above)
Comment (nielslange) [reply plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts]: Good catch. I think I've initially added that part as after installing a different language within WP-Admin, I always go to the update page and update the translations. That said, as we're running `wp language plugin install woocommerce nl_NL` immediately before `wp language plugin update woocommerce`, I think the later command is indeed not necessary. I'll quickly try that.

File: plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts (hunk 2 above)
Comment (nielslange) [reply plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts]: See https://github.com/woocommerce/woocommerce/pull/45371#discussion_r1516018178

**Comment by nielslange** [comment]
Thanks for reviewing and pre-approving this PR, @gigitux. I've updated the PR based on your feedback and as all tests are passing, I'm merging it now.

**Pull request merged**